include/rocksdb/utilities/cache_dump_load.h

```diff
@@ -0,0 +1,82 @@
+//  Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#pragma once
+
+#include <set>
+
+#include "rocksdb/cache.h"
+#include "rocksdb/env.h"
+#include "rocksdb/file_system.h"
+#include "rocksdb/io_status.h"
+#include "rocksdb/secondary_cache.h"
+#include "rocksdb/table.h"
+
+namespace ROCKSDB_NAMESPACE {
+
+static const int kCacheDumpMajorVersion = 0;
+static const int kCacheDumpMinorVersion = 1;
+
+class CacheDumpWriter {
```

Comment:
We should add some basic comments here on the purpose (allow user to customize how cache data is transported from source to destination), or at least mark EXPERIMENTAL / SUBJECT TO CHANGE.

Reply:
Sure, I will add the some explanation about how users can customize the writer and transmit the blocks. But it is still a good iead to mark it as EXPERIMENTAL